tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -15,7 +15,8 @@ def test_hyperopt_parameters(use_hyperband):
 
     sampler = TPESampler(**TPESampler.hyperopt_parameters())
     study = optuna.create_study(
-        sampler=sampler, pruner=optuna.pruners.HyperbandPruner() if use_hyperband else None
+        sampler=sampler,
+        pruner=optuna.pruners.HyperbandPruner(max_resource=50) if use_hyperband else None,
```

Comment:
Would you mind motivating this change?

Reply:
The previous changes were mixed in. I will put it back.